docs/concepts/registries.md

```diff
@@ -8,7 +8,7 @@ Most things that require registration in the game are handled by the Forge regis
 Registering Things
 ------------------
 
-The recommended way to register things is through the `RegistryEvent`s. These [events][] are fired right after preinitialization, In `RegistryEvent.NewRegistry`, registries should be created. Later, `RegistryEvent.Register` is fired once for each registered registry. Because `Register` is a generic event, the event handler should set the type parameter to the type of the object being registered. The event will contain the registry to register things to (`getRegistry`), and things may be registered with `register` (or `registerAll`) on the registry. Here's an example of an event handler that registers blocks:
+The recommended way to register things is through the `RegistryEvent`s. These [events][] are fired after mod constructors are called and before configs are loaded, In `RegistryEvent.NewRegistry`, registries should be created. Later, `RegistryEvent.Register` is fired once for each registered registry. Because `Register` is a generic event, the event handler should set the type parameter to the type of the object being registered. The event will contain the registry to register things to (`getRegistry`), and things may be registered with `register` (or `registerAll`) on the registry. Here's an example of an event handler that registers blocks:
```

Comment:
"... before configs are loaded, In `RegistryEvent.NewRegistry`..." should be the end/beginning of a sentence, not a comma.